prime-router/src/test/java/com/function/FunctionTest.java

```diff
@@ -0,0 +1,53 @@
+package com.function;
+
+import com.microsoft.azure.functions.*;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+
+import java.util.*;
+import java.util.logging.Logger;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.*;
+
```

Comment:
Love the use of mocks

Reply:
Removed - generated automatically by VSCode